terms.html

```diff
@@ -51,9 +51,12 @@
   </dd>
   <dt><dfn data-lt="entities|entity's">entity</dfn></dt>
   <dd>
-A thing with distinct and independent existence, such as a person,
-organization, or device that performs one or more roles in the ecosystem.
-  </dd>
+Anything that can be referenced in statements as an abstract or concrete noun.
```

Comment:
This seems more correct to me, but no objection if we move forward with "as":

```suggestion
Anything that can be referenced in statements by an abstract or concrete noun.
```

Reply:
I think either can be used. Alternatively --
```suggestion
Anything that an abstract or concrete noun in a statement can reference.
```

Reply:
I have to merge this now... I'm going with the original as it feels more clear to me.